stream.go

```diff
@@ -237,6 +237,11 @@ func (st *Stream) produceKVs(ctx context.Context, threadId int) error {
 				continue
 			}
 
+			// Do not send the keys which are banned.
+			if st.db.isBanned(item.Key()) {
```

Comment:
Maybe move this above choose key.

Reply:
This is no longer needed with the iterator changes.